src/common/servlist.c

```diff
@@ -201,6 +201,9 @@ static const struct defaultserver def[] =
 	{"Hashmark",	0},
 	{0,			"irc.hashmark.net"},
 
+	{"ICQ-Chat", 0},
```

Comment:
Update this to include that it supports SASL and SSL like others.

Reply:
Updated